src/React/doc/index.rst

```diff
@@ -98,16 +98,38 @@ Using with AssetMapper
 Because the JSX format isn't pure JavaScript, using this library with AssetMapper
 requires some extra steps.
 
-#. Compile your ``.jsx`` files to pure JavaScript files. This can be done by
+.. note:: 
+
+    Compile your ``.jsx`` files to pure JavaScript files. This can be done by
    installing Babel and the ``@babel/preset-react`` preset. Example:
    https://github.com/symfony/ux/blob/2.x/ux.symfony.com/package.json
 
-#. Point this library at the "built" controllers directory that contains the final
+1- Create package.json in your project. This file is not included in the **Standard package**. (code below)
+
+.. code-block:: json-object
+
+ {
+   "devDependencies": {
+    "@babel/cli": "^7.21.5",
+    "@babel/core": "^7.21.8",
+    "@babel/preset-react": "^7.18.6",
+    "svelte": "^3.59.1"
+  },
+  "license": "MIT",
+  "private": true,
+  "scripts": {
+    "build": "yarn build-react && yarn build-svelte",
+    "build-react": "babel assets/react --out-dir assets/build/react --presets=@babel/preset-react",
+    "build-svelte": "node bin/compile_svelte.js"
+  }
+ }
+
```

Comment:
I think we don't need to add the content of the package.json. Let's stay simple and say that we need a package.json and to install few dependencies.